app/controllers/dialogs_view_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# A controller to show registered dialogs.
+class DialogsViewController < ActionController::Base
+  PER = 10
```

Comment:
もう少し親切に
```suggestion
  ITEMS_PER_PAGE = 10
```
とかにしましょうか？

Reply:
定数名を`PER`→ `ITEMS_PER_PAGE`にしました。